conans/test/util/tools_test.py

```diff
@@ -993,6 +997,36 @@ def detect_windows_subsystem_test(self):
         else:
             self.assertEqual(str, type(result))
 
+    @attr('slow')
+    def get_filename_download_test(self):
+        # Create a tar file to be downloaded from server
+        with tools.chdir(tools.mkdir_tmp()):
+            import tarfile
+            tar_file = tarfile.open("sample.tar.gz", "w:gz")
+            tools.mkdir("test_folder")
+            tar_file.add(os.path.abspath("test_folder"), "test_folder")
+            tar_file.close()
+            file = os.path.abspath("sample.tar.gz")
+            assert (os.path.exists(file))
+
+        # Instance stoppable thread server and add endpoint
+        thread = StoppableThreadBottle(port=8266)
```

Comment:
Change the default to this one for example, the 8080 is too much typical and potentially busy